### DISCUSSION TITLE
How to add dependencies?

### DISCUSSION THREAD

**Discussion by Myzel394** [cookiecutter/cookiecutter-django#3730] [Q&A] [answered]
I can't add dependencies by adding them in the `requirements/base.txt`. How do I add them properly? I'm using PyCharm with the remote interpreter setup enabled.

**Comment by browniebroke** [comment, marked as answer]
Since this question was upvoted by a few people and there was a [duplicate question](https://github.com/cookiecutter/cookiecutter-django/discussions/4084) asked today, here it is (assuming you're using Docker):

1. Add dependency to the relevant requirement file (`base.txt`, `local.txt` or `production.txt`)
2. Rebuild the relevant docker images
3. Restart the containers

if you're not using Docker:

1. Add dependency to the relevant requirement file (`base.txt`, `local.txt` or `production.txt`)
2. In your virtualenv, run `pip install -r local.txt`
3. Restart your Django server (and Celery, if apropriate)
